src/util/circle_static.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020  Patryk Seregiet
+ *
+ * This file is part of ReplaySorcery.
+ *
+ * ReplaySorcery is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ReplaySorcery is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with ReplaySorcery.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "circle_static.h"
+#include "memory.h"
+
+void rsCircleStaticCreate(RSCircleStatic *circle, size_t size) {
+   circle->data = rsMemoryCreate(size);
+   circle->size = size;
+   circle->index = 0;
+}
+
+void rsCircleStaticDestroy(RSCircleStatic *circle) {
+   if (circle->data) {
```

Comment:
```suggestion
   if (circle->data != NULL) {
```
Theres probably more but these are the ones I could find just so you can press a button in github to add them in.